imagekitio/models/results/EmbeddedMetadata.py

```diff
@@ -1,6 +1,6 @@
 class EmbeddedMetadata:
-    def __init__(self, x_resolution=None, y_resolution=None, date_created=None, date_timecreated=None):
+    def __init__(self, x_resolution=None, y_resolution=None, date_created=None, date_time_created=None):
         self.x_resolution = x_resolution
         self.y_resolution = y_resolution
         self.date_created = date_created
-        self.date_timecreated = date_timecreated
```

Comment:
This is the complete structure of embedded metadata - https://docs.imagekit.io/api-reference/metadata-api#metadata-object-structure
This has not changed pretty much in the past but there is a possibility of introducing new fields. Should we make Python SDK in a way that it is not prone to such changes on the backend? 
This can be taken up separately as I believe the same applies to other APIs as well e.g. list files being one of them. The end goal is that SDK should not break if the backend API introduces a non-breaking change i.e. introduce a new field in the response.